src/webgpu/api/validation/resource_usages/textureUsageInPassEncoder.spec.ts

```diff
@@ -854,12 +876,9 @@ g.test('bindings_in_bundle')
       success = true;
     }
 
-    if (type0 === 'writeonly-storage-texture' && type1 === 'writeonly-storage-texture') {
-      success = true;
-    }
+    if (t.isWritableBinding(type0) && t.isWritableBinding(type1)) success = true;
```

Comment:
I thought the ONLY write-write conflict allowed was between two writeonly-storage-textures (not between writeonly-storage-texture and arbitrary other write usages. (Though it probably also would be allowed between readwrite-storage-texture and writeonly-storage-texture in the future.)

We may eventually want to try to find a way to describe the [compatible usage list](https://gpuweb.github.io/gpuweb/#compatible-usage-list) in capability_info, but that's not necessary now (and may never be necessary).

Reply:
Ack. We have already had tests for combinations for binding types + render-target, it is kind of like a compatible usage list.